test/test_rb.py

```diff
@@ -32,6 +33,247 @@
 }
 
 
+@pytest.mark.parametrize(
```

Comment:
note to self: to review this file later

Reply:
FYI - these tests are pretty much identical to the other tests in `test_rb.py` that were there before my changes. The only difference is that the ReplayBuffers are from `torchrl.data.replay_buffers.rb_prototype` instead of `torchrl.data.replay_buffers.replay_buffers` and there are some new methods that didn't use to exist like `update_tensordict_priority`.